src/styles/_catalog.scss

```diff
@@ -1,22 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
```

Comment:
you don't need this with `grid`

Reply:
I changed it in my code and I checked it here - the code is right. Please review it.